tests/e2e/workloads/amq/test_rgw_kafka_notifications.py

```diff
@@ -147,4 +151,27 @@ def test_rgw_kafka_notifications(self, bucket_factory):
                 "RGW bucket notification is not working as expected."
             )
 
+        # Validate the timestamp events
+        cmd = (
+            f"bin/kafka-console-consumer.sh --bootstrap-server {constants.KAFKA_ENDPOINT} "
+            f"--topic {self.kafka_topic.name} --from-beginning --timeout-ms 20000"
+        )
+        pod_list = get_pod_name_by_pattern(
+            pattern="my-cluster-zookeeper", namespace=constants.AMQ_NAMESPACE
+        )
+        zookeeper_obj = get_pod_obj(name=pod_list[0], namespace=constants.AMQ_NAMESPACE)
+        event_obj = zookeeper_obj.exec_cmd_on_pod(command=cmd)
+        log.info(f"Event obj: {event_obj}")
+        event_time = event_obj.get("Records")[0].get("eventTime")
```

Comment:
Should we check for the specific timestamp pattern only from 4.8 onwards?

Reply:
Added condition